client/cypress/integration/visualizations/box_plot_spec.js

```diff
@@ -58,11 +58,11 @@ describe('Box Plot', () => {
       'BoxPlot.YAxisLabel': 'Y Axis',
     });
 
-    // Wait for proper initialization of visualization
-    cy.wait(500); // eslint-disable-line cypress/no-unnecessary-waiting
-
     cy.getByTestId('VisualizationPreview').find('svg').should('exist');
 
-    cy.percySnapshot('Visualizations - Box Plot');
+    cy.viewport(1280, 1024);
+    cy.wait(500); // eslint-disable-line cypress/no-unnecessary-waiting
+    cy.screenshot();
+    cy.percySnapshot('Visualizations - Box Plot', { widths: [1280] });
```

Comment:
Oh I thought you were gonna go for:
`cy.percySnapshot('Visualizations - Box Plot', { widths: [1000] });`

Reply:
There is `cy.viewport` few lines above - I want to take snapshots of the same sizes that we have for other tests (1280 and 375px). But ATM I'm just checking if it will work - if yes, I'll improve it

Reply:
You could also set `cypress.json` to `viewportWidth: 1000` so no percy shots get invalidated.

Reply:
`1280px` seems a better number :slightly_smiling_face: (since the Cypress viewport is defined prior to it, should make no difference). In case it works it's probably better to set it globally for Cypress.